### PR TITLE
add virstrain recipes

### PR DESCRIPTION
**Pull request by liaoherui**
Describe your pull request here
Add : VirStrain - An RNA/DNA virus strain-level identification tool for short reads.
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by liaoherui** [comment]
@BiocondaBot please add label

**Comment by liaoherui** [comment]
@BiocondaBot please add label

File: recipes/virstrain/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  license: MIT
+  license_family: OTHER
+  license_file: 
+  summary: An RNA/DNA virus strain-level identification tool for short reads.
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - your-github-id-here
```
Comment (simonbray) [inline recipes/virstrain/meta.yaml:57]:
```suggestion
    - liaoherui
```
? :)

File: recipes/virstrain/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  imports:
+    - VirStrain
+  commands:
+    - virstrain --help
+    - virstrain_build --help
+
+about:
+  home: https://github.com/liaoherui/VirStrain
+  license: MIT
+  license_family: OTHER
```
Comment (simonbray) [inline recipes/virstrain/meta.yaml:49]:
```suggestion
  license_family: MIT
```

File: recipes/virstrain/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  license_file: 
+  summary: An RNA/DNA virus strain-level identification tool for short reads.
+  doc_url: 
+  dev_url:
```
Comment (simonbray) [inline recipes/virstrain/meta.yaml:53]:
```suggestion
  summary: An RNA/DNA virus strain-level identification tool for short reads.
```
better to remove these blank entries

File: recipes/virstrain/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+requirements:
+  host:
+    - biopython ==1.74
+    - matplotlib-base ==3.1.2
+    - networkx ==2.4
+    - numpy ==1.17.3
+    - pandas ==1.0.1
+    - pip
+    - plotly ==3.10.0
+    - python ==3.7.3
```
Comment (simonbray) [inline recipes/virstrain/meta.yaml:29]:
```suggestion
    - pip
    - python
```
you should only need python and pip in the host section

File: recipes/virstrain/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  run:
+    - biopython ==1.74
+    - matplotlib-base ==3.1.2
+    - networkx ==2.4
+    - numpy ==1.17.3
+    - pandas ==1.0.1
+    - plotly ==3.10.0
+    - python ==3.7.3
```
Comment (simonbray) [inline recipes/virstrain/meta.yaml:37]: can you remove pins where possible?

File: recipes/virstrain/meta.yaml (hunk 1 above)
Comment (liaoherui) [reply recipes/virstrain/meta.yaml]: Fixed, thanks!

File: recipes/virstrain/meta.yaml (hunk 2 above)
Comment (liaoherui) [reply recipes/virstrain/meta.yaml]: Fixed, thanks!

File: recipes/virstrain/meta.yaml (hunk 3 above)
Comment (liaoherui) [reply recipes/virstrain/meta.yaml]: Fixed, thanks!

File: recipes/virstrain/meta.yaml (hunk 4 above)
Comment (liaoherui) [reply recipes/virstrain/meta.yaml]: Fixed, thanks!

**Review by simonbray — APPROVED** [review]

**Comment by simonbray** [comment]
@BiocondaBot please merge

**Pull request merged**